src/lib/components/ui/select/select.svelte

```diff
@@ -101,7 +118,8 @@
 			class:hover
 			class:active
 			class:focus
-			class:focused
+			class:focused={input?.classList.contains('focused')}
```

Comment:
This is quite weird. So you are setting the `focused` class if this element already has a `focused` class.